src/components/policy/policy_external/src/policy_manager_impl.cc

```diff
@@ -301,17 +310,121 @@ void FilterInvalidParameters(policy_table::RpcParameters& rpc_parameters) {
   rpc_parameters.parameters->swap(valid_params);
 }
 
+/**
+ * @brief FilterInvalidNotifications filter notification priorities that are not
+ * present in schema
+ * @param notification priorities to filter
+ */
+void FilterInvalidNotifications(
+    policy_table::NumberOfNotificationsPerMinute& notifications) {
+  policy_table::NumberOfNotificationsPerMinute valid_notifications;
+  for (const auto& notification : notifications) {
+    policy_table::Priority priority;
+    if (policy_table::EnumFromJsonString(notification.first, &priority)) {
+      valid_notifications.insert(notification);
+    }
+  }
+  notifications.swap(valid_notifications);
+}
+
+/**
+ * @brief FilterInvalidApplicationParameters filter app params that are not
+ * present in schema
+ * @param object of app policy params to filter
```

Comment:
@JackLivio don't forget to add parameter name as the second argument of doxygen comment. Like:
```
@param app_params object of app policy params to filter
```